### PR TITLE
For ubuntu pass the packages directly by name.  Ubuntu does not know

### PR DESCRIPTION
**Pull request by dvalinrh**
about dnf, and there are multiple php matches.

### CODE REVIEW

File: phoronix/run_phoronix.sh [hunk 1]
```diff
@@ -119,16 +119,26 @@ if [ $show_usage -eq 1 ]; then
...
+       	*)
+		for pkg in cli common php; do
+			package=`dnf list | grep  php  | grep $pkg | awk '{print $1}'`
+			if [[ $package == "" ]]; then
+				error_out "Did not find a php $pkg package" 1
+			fi
+			test_tools/package_tool --packages $package
+			if [[ $? != "0" ]]; then
+				error_out "Failed to install one or more $php_cli,$php_common,$php_xml" 1
+			fi
```
Comment (kdvalin) [inline phoronix/run_phoronix.sh:138]: Why 2 `test_tools/package_tool` calls, when you can set something like the `packages_to_install` variable and do 1 call with an error check?

**Review by kdvalin — CHANGES_REQUESTED** [review]

File: phoronix/run_phoronix.sh [hunk 2]
```diff
@@ -119,16 +119,23 @@ if [ $show_usage -eq 1 ]; then
...
+			if [[ $packages == "" ]]; then
+				error_out "Did not find a php $pkg package" 1
+			fi
+		done
+	;;
+esac
+test_tools/package_tool --packages "php-cli,php,php-common"
+if [[ $? != "0" ]]; then
+	error_out "Failed to install one or more php_cli,php_common,php_xml" 1
+fi
```
Comment (kdvalin) [inline phoronix/run_phoronix.sh:138]: If you're going to install php-cli,php,php-common, why do you need the `case`?

Also why are we installing packages in the wrapper?  Shouldn't we be using the configuration files in Zathras for this?

**Review by kdvalin — CHANGES_REQUESTED** [review]

File: phoronix/run_phoronix.sh (hunk 1 above)
Comment (dvalinrh) [reply phoronix/run_phoronix.sh]: We can move those installs to the config files,however part of the problem faced here has been names changing on those packages every so often.

File: phoronix/run_phoronix.sh (hunk 2 above)
Comment (dvalinrh) [reply phoronix/run_phoronix.sh:138]: Think about it, you folks are right, lets move the packages to the configs.  We can deal with the package names changing as need be.

**Review by kdvalin — APPROVED** [review]
LGTM, open another PR in Zathras to include the package names in the config.

**Pull request merged**